lnd.go

```diff
@@ -689,12 +668,7 @@ func getTLSConfig(tlsCertPath string, tlsKeyPath string, tlsExtraIPs,
 
 	}
 
-	tlsCfg := &tls.Config{
-		Certificates: []tls.Certificate{certData},
-		CipherSuites: tlsCipherSuites,
-		MinVersion:   tls.VersionTLS12,
-	}
-
+	tlsCfg := cert.TLSConfFromCert(certData)
```

Comment:
Also here it may be easier to review if the refactor is separate from the move. But this is a small commit, so not really a problem.